core/orch_test.go

```diff
@@ -387,13 +417,13 @@ func TestTranscoderManagerTranscoding(t *testing.T) {
 
 	// fatal error should retry and remove from list
 	s.SendError = fmt.Errorf("SendError")
-	_, err = m.Transcode(&SegTranscodingMetadata{})
+	_, err = m.Transcode(&SegTranscodingMetadata{AuthToken: &net.AuthToken{SessionId: testSessionId}})
```

Comment:
Let's also assert the first return value to be nil

Reply:
That would return the transcoded data correct? I think you meant NotNil? Adding it in.